.rubocop_todo.yml

```diff
@@ -87,16 +87,6 @@
 #    - 'spec/support/flow_helpers.rb'
 #    - 'spec/support/xml_extractor.rb'
 
-# Offense count: 8
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle, BlockForwardingName.
-# SupportedStyles: anonymous, explicit
-Naming/BlockForwarding:
```

Comment:
does this style come from `govuk-rubocop`, I personally prefer the non-anonymous block forwarding style

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/BlockForwarding.

could just set `EnforcedStyle: explicit`? don't know how others feel